openslides_backend/action/actions/user/create_update_permissions_mixin.py

```diff
@@ -213,8 +213,8 @@ def check_permissions(self, instance: dict[str, Any]) -> None:
         """
         self.assert_not_anonymous()
 
-        if "forwarding_committee_ids" in instance:
-            raise PermissionDenied("forwarding_committee_ids is not allowed.")
+        # if "forwarding_committee_ids" in instance:
```

Comment:
remove code and document in action descriptions, if you really want

Reply:
see comment in user.create for why the documentation wasn't amended